src/ripple/net/impl/RegisterSSLCerts.cpp

```diff
@@ -60,7 +59,7 @@ registerSSLCerts(
     if (!store)
     {
         ec = boost::system::error_code(
-            safe_cast<int>(::ERR_get_error()),
+            static_cast<int>(::ERR_get_error()),
```

Comment:
Looking at this again, would it be worth adding a range-check `assert` like you did with STLedgerEntry?

Reply:
I'm hesitant to put in an assert since this is trying to create an `error_code` that presumably some other code wants to handle, instead of terminate.  I've translated the out of range result into an `ERANGE`.  What do you think about that?